ci-operator/config/openshift/release/openshift-release-master__ocp-4.7.yaml

```diff
@@ -27,6 +27,21 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: e2e-z-upi
+  cron: '@yearly'
+  steps:
+    env:
+      http_proxy: 127.0.0.1
+      https_proxy: 127.0.0.1
```

Comment:
This is not that simple - this needs to go into the step, otherwise this has the "set these parameters expected by the step" semantics, see the pj-rehearse failure:

```
time="2020-11-18T17:31:49Z" level=error msg="Failed to get resolved config for test" error="failed resolve ReleaseBuildConfiguration: Failed resolve MultiStageTestConfiguration: [e2e-z-upi: no step declares parameter \"http_proxy\", e2e-z-upi: no step declares parameter \"https_proxy\"]" org=openshift repo=release 
```

Reply:
Yeah, I saw that :|